skt/executable.py

```diff
@@ -19,16 +19,18 @@
 import ast
 import datetime
 import json
-import junit_xml
 import logging
 import os
 import shutil
 import sys
 import time
+
+import junit_xml
+
+import skt
 import skt.publisher
 import skt.reporter
 import skt.runner
-import skt
```

Comment:
I'm curious, what's the logic behind the import reordering you're doing here and other files? Could you perhaps put those into a separate commit and elaborate there?

Reply:
That is being done to please `flake8`. There are ordering and spacing requirements for modules that come from core python, extra modules installed later, and the skt project itself.

Reply:
Wow, that's heavy. And I thought *I* was a perfectionist :smiley:.